crates/modelardb_compression/src/models/timestamps.rs

```diff
@@ -327,17 +331,93 @@ mod tests {
             1579694400353,
             1579694400493,
             1579694400650,
-        ]);
+        ], Some(4));
+    }
+
+    #[test]
+    fn compress_and_decompress_bucket_sized_1_timestamps_for_an_irregular_time_series() {
```

Comment:
I am not sure this is an universal convention in Rust but it seems that many of our other tests always have "test_" as a prefix in the test function name. If this is a convention we want to use then it should be changed in the tests in this file.

Reply:
Good catch, I have fixed it for all of the tests in this file.